src/simplejavatexteditor/SupportedKeywords.java

```diff
@@ -33,8 +33,8 @@
                 "template", "typename", "class", "friend", "private", "this", "using", "const_cast",
                 "inline", "public", "throw", "virtual", "delete", "mutable", "protected", "true", "wchar_t" };
 
-    public String[] getSupportedLangage() {
-        return supportedLangage;
+    public String[] getsupportedLanguage() {
```

Comment:
As long as you are here, you could also change its name to be plural ;)
`public String[] getsupportedLanguages() {`

Reply:
@LordStephen77 Did you get a chance to look at my comment